tests/offtarget/test_bwa.py

```diff
@@ -68,6 +73,19 @@ def test_hit_build_rc() -> None:
     assert hit_r.negative is True
 
 
+def test_header_is_properly_constructed(ref_fasta: Path) -> None:
+    """Tests that bwa will return a properly constructed header."""
+    with BwaAlnInteractive(ref=ref_fasta, max_hits=1) as bwa:
+        header: SamHeaderType = bwa.header.to_dict()
+        assert set(header.keys()) == {"HD", "SQ", "PG"}
+        assert header["HD"] == {"GO": "query", "SO": "unsorted", "VN": "1.5"}
+        assert header["SQ"] == [{"LN": 10001, "SN": "chr1"}]
+        assert len(header["PG"]) == 1
+        program_group: dict[str, str] = cast(list[dict[str, str]], header["PG"])[0]
+        program_group["ID"] = "bwa"
+        program_group["PN"] = "bwa"
```

Comment:
We should probably update `bwa-aln-interactive` to use the new executable name...

https://github.com/fulcrumgenomics/bwa-aln-interactive/issues/5